packages/components/src/styles/table-cell/table-cell.stories.ts

```diff
@@ -41,580 +41,190 @@ export const Default = {
 };
 
 /**
- * These examples demonstrate the usage of sd-table-cell in various contexts.
- * The examples are intended solely for illustrating how sd-table-cell can be used to style tables.
- * The data generation and table sorting logic should not be used in production environments.
+ * Use the `&--bg-*` class for alternative appearances.
```

Comment:
Show it like this, pls

![image](https://github.com/user-attachments/assets/70f1c299-b19a-41bf-8000-4668471ff165)